modules/cloud-run-v2/service.tf

```
@@ -0,0 +1,232 @@
+            container_port = ports.value.container_port
+            name           = ports.value.name
+          }
+        }
+        dynamic "volume_mounts" {
+          for_each = coalesce(containers.value.volume_mounts, tomap({}))
+          content {
+            name       = volume_mounts.key
+            mount_path = volume_mounts.value
+          }
+        }
+        dynamic "liveness_probe" {
+          for_each = containers.value.liveness_probe == null ? [] : [""]
+          content {
+            initial_delay_seconds = containers.value.liveness_probe.initial_delay_seconds
+            timeout_seconds       = containers.value.liveness_probe.timeout_seconds
+            period_seconds        = containers.value.liveness_probe.period_seconds
+            failure_threshold     = containers.value.liveness_probe.failure_threshold
+            dynamic "http_get" {
+              for_each = containers.value.liveness_probe.http_get == null ? [] : [""]
+              content {
+                path = containers.value.liveness_probe.http_get.path
+                dynamic "http_headers" {
+                  for_each = coalesce(containers.value.liveness_probe.http_get.http_headers, tomap({}))
+                  content {
+                    name  = http_headers.key
+                    value = http_headers.value
+                  }
+                }
+              }
+            }
+            dynamic "grpc" {
+              for_each = containers.value.liveness_probe.grpc == null ? [] : [""]
+              content {
+                port    = containers.value.liveness_probe.grpc.port
+                service = containers.value.liveness_probe.grpc.service
+              }
+            }
+          }
+        }
+        dynamic "startup_probe" {
+          for_each = containers.value.startup_probe == null ? [] : [""]
+          content {
+            initial_delay_seconds = containers.value.startup_probe.initial_delay_seconds
+            timeout_seconds       = containers.value.startup_probe.timeout_seconds
+            period_seconds        = containers.value.startup_probe.period_seconds
+            failure_threshold     = containers.value.startup_probe.failure_threshold
+            dynamic "http_get" {
+              for_each = containers.value.startup_probe.http_get == null ? [] : [""]
+              content {
+                path = containers.value.startup_probe.http_get.path
+                dynamic "http_headers" {
+                  for_each = coalesce(containers.value.startup_probe.http_get.http_headers, tomap({}))
+                  content {
+                    name  = http_headers.key
+                    value = http_headers.value
+                  }
+                }
+              }
+            }
+            dynamic "tcp_socket" {
+              for_each = containers.value.startup_probe.tcp_socket == null ? [] : [""]
+              content {
+                port = ontainers.value.startup_probe.tcp_socket.port
```

```suggestion
                port = containers.value.startup_probe.tcp_socket.port
```
how did this make it through?

---

Sharp eye! I think that dynamic sections are not evaluated if not needed, and as we don't have an example to test this, this fell through.